frontend/src/static/js/v_authorship.js

```diff
@@ -171,6 +181,9 @@ window.vAuthorship = {
           const out = {};
           out.path = file.path;
           out.lineCount = lineCnt;
+          out.active = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
+          out.wasCodeLoaded = false;
+          out.isLoading = true;
```

Comment:
Perhaps `shouldLoad` / `shouldRender` may be a better variable name instead of `isLoading`?